GitDepend.UnitTests/Commands/AddCommandTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitDepend.Busi;
+using GitDepend.CommandLine;
+using GitDepend.Commands;
+using GitDepend.Configuration;
+using Microsoft.Practices.Unity;
+using NUnit.Framework;
+using Telerik.JustMock;
+using Telerik.JustMock.Helpers;
+
+namespace GitDepend.UnitTests.Commands
+{
+    [TestFixture]
+    public class AddCommandTests : TestFixtureBase
+    {
+        private IFileSystem _fileSystem;
+        private const string GitDependConfigFile = "GitDepend.json";
+
+        [SetUp]
+        public void AddCommandTestsSetup()
+        {
+            _fileSystem = RegisterMockFileSystem();
+            EnsureDirectory(_fileSystem, Lib1Directory);
+            EnsureDirectory(_fileSystem, Lib2Directory);
+            EnsureFiles(_fileSystem, Lib2Directory, new List<string>()
+            {
+                GitDependConfigFile
+            });
+        }
+
+
+        [Test]
+        public void AddCommand_Succeeds()
+        {
+            var options = new AddSubOptions()
+            {
+                Directory = Lib2Directory,
+                Branch = "master",
+                DependencyDirectory = "..\\lib3",
+                Url = "https://test-url.com/repos/lib3"
+            };
+
+            var command = new AddCommand(options);
+
+            var code = command.Execute();
+
+            Assert.AreEqual(ReturnCode.Success, code);
```

Comment:
It looks like you are only ever testing the return code. There should also be a test to see that the config file was actually modified (or not modified) in the expected way.